minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoEnvServer.java

```diff
@@ -690,7 +690,7 @@ private void peek(String command, Socket socket, DataInputStream din) throws IOE
     public byte[] getObservation(boolean done)  {
         byte[] obs = envState.obs;
         if (obs == null){
-            
+            return new byte[0];
```

Comment:
what does this do?

Reply:
This is needed when we disabled rendering - it seems it causes an error to appear elsewhere when the POV is missing